index.html

```diff
@@ -170,13 +162,12 @@ <h6 class="art-print">Profesional Art Printing Data</h6>
               A daily selection of privately personalized reads; no accounts or
               sign-ups required. has been the industry's standard
             </p>
+            <ul class="pro-tag">
+              <li class="pro-h">html</li>
+              <li class="pro-h">bootstrap</li>
+              <li class="pro-h">Ruby</li>
+            </ul>
           </div>
-          <ul class="pro-tag">
-            <li class="pro-h">html</li>
-            <li class="pro-h">bootstrap</li>
-            <li class="pro-h">Ruby</li>
-          </ul>
-          <button class="see-proj" type="button">See Project</button>
         </div>
       </section>
       <section class="about">
```

Comment:
- Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Aligning this section's elements to match the Figma design.
   - [ ] Adjusting the spacing between the background image and the text so they don't overlap.
   - [ ] Adjusting the width of the horizontal line to match the Figma design.

| Your design | Figma design | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160218331-d6f44065-b1db-48c6-875d-df18714af5bd.png) | ![image](https://user-images.githubusercontent.com/24703171/160218350-e7455eaf-a097-4d18-86cb-6570052c97fe.png) |

Reply:
@gemmen29 Please how am i supposed to maintain the same width if you ask i use a template of 768px as against the figma template of 1440px?